test/fixtures/object-assign/case-1/after.js

```diff
@@ -0,0 +1,21 @@
+const assert = require('assert');
+
+const res = Object.assign({});
```

Comment:
I pulled these test cases from the ponyfill repo's README. Ι can add more test cases if there are others we want to test for

Reply:
this looks fine for now :) most of the actual usage of these packages is pretty straightforward, we can improve later